ArduPlane/quadplane.h

```diff
@@ -608,6 +608,7 @@ class QuadPlane
         FS_RTL=(1<<20),
         DISARMED_TILT_UP=(1<<21),
         SCALE_FF_ANGLE_P=(1<<22),
+        SUPPRESS_YAW_TRANSITION=(1<<23),
```

Comment:
I think bit 22 (23 bits) is the limit of the mantissa. The problem is float transport of parameters over MAVLink. I guess we could add a auto test for it. But I would expect at some combination of high bits the lower bits start getting lost.